PLog.cs

```diff
@@ -1,13 +1,35 @@
-﻿using System;
-
-namespace p2pcopy
-{
-	public class PLog
-	{
-		public static void DEBUG(string fmt, params object[] args)
-		{
-			//Console.WriteLine(string.Format(fmt, args));
-		}
-	}
-}
-	
\ No newline at end of file
+﻿using System;
+
+namespace p2pcopy
+{
+    public class PLog
+    {
+        public static bool debug = false;
+        public static bool verbose = false;
+
+        public static void DEBUG(string fmt, params object[] args)

```

Comment:
cosmetic: AFAIK Microsoft .NET API design guidelines say: use PascalCase, not UPPERCASE, and use PascalCase for public fields, camelCase for private ones (maybe debug & verbose can be made private?).

Reply:
(if not private, internal)

Reply:
Done!